setup.py

```diff
@@ -63,6 +63,7 @@
                'pint>=0.8',
                'statsmodels',
                'mrcz>0.3.4',
+               'sparse',
```

Comment:
Thinking about it, this can also be an issue to install in Debian. Also, sparse is not yet in conda forge https://github.com/pydata/sparse/issues/115 .Hence, I'll make lazy optional.